src/mdlComponentHandler.js

```diff
@@ -51,19 +51,6 @@ var componentHandler = (function() {
   }
 
   /**
-   * Returns true if the given element has already been upgraded for the given
-   * class.
-   * @param {HTMLElement} element The element we want to check.
-   * @param {string} jsClass The class to check for.
-   * @return boolean
-   * @private
-   */
-  function isElementUpgraded_(element, jsClass) {
```

Comment:
I'd rather we have that for code readability over removing it and creating less readable code internally.

Let's be honest, the overhead that may be caused by keeping the method is nothing to worry about at the moment. Seems like premature micro-optimization otherwise.